src/Mpociot/ApiDoc/Generators/AbstractGenerator.php

```diff
@@ -267,7 +267,8 @@ protected function parseRule($rule, $ruleName, &$attributeData, $seed)
                 break;
             case 'in':
                 $attributeData['description'][] = Description::parse($rule)->with($this->fancyImplode($parameters, ', ', ' or '))->getDescription();
-                $attributeData['value'] = $faker->randomElement($parameters);
+//                $attributeData['value'] = $faker->randomElement($parameters);
```

Comment:
Hi @prodigy7 thanks for the PR. I'm not sure I understand why we're always choosing the first parameter. Maybe you should add a code comment for anyone who sees it in the future.

Also, please don't comment out old code. Delete it. Let Git do the tracking. 😀

Reply:
I'm not very familiar with cURL, but the rest of the PR looks okay. So, if you attend to this, I can merge this PR. Also, can you update the documentation to mention the `env` variables it uses?